app/Repositories/Eloquents/BotRepository.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace App\Repositories\Eloquents;
+
+use Auth;
+use App\Models\Bot;
+use App\Repositories\Eloquents\BaseRepository;
+use App\Repositories\Interfaces\BotRepositoryInterface;
+
+class BotRepository extends BaseRepository implements BotRepositoryInterface
+{
+    public function getModel()
+    {
+        return Bot::class;
+    }
+
+    public function getAllByUser()
+    {
+        return Auth::user()->bots()
+                            ->orderBy('bots.created_at', 'desc')
+                            ->get();
+    }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BotRepository.php</br>**Line**: 23</br>The closing brace for the class must go on the next line after the body